proposed/2023/project-exclusion-for-transitive-dependencies.md

```diff
@@ -0,0 +1,130 @@
+# Support excluding projects from transitive dependencies
+
+- Author Name [Greybird](https://github.com/Greybird)
+- Start Date 2023-07-11
+- GitHub Issue [12585](https://github.com/NuGet/Home/issues/12585)
+- GitHub PR https://github.com/NuGet/NuGet.Client/pull/5302
+
+## Summary
+
+Currently, when listing transitive packages using `dotnet list package --include-transitive`, both project and packages dependencies are reported. The goal of this proposal is to provide a way to filter the projects out of the report.
+
+## Motivation 
+
+Packages and projects are two vastly different types of references.
+Packages are published to a repository, allowing consumption by multiple pieces of software and repositories. Metadata are attached to packages or package versions, either included in the package, or added as a decoration - for deprecation or vulnerabilities for example -, which can be consumed to take decisions. On the contrary, projects are _local_ entities, living in a single repository, and do not have an existence outside of it.
```

Comment:
This paragraph is probably too strong of a statement. 
Projects and Packages are incredibly close. When you use PackageReference, and you pack a project, by default, all the project references *become* package references. 

The NuGet resolver treats these equivalently. Packages can be preferred over projects if the situation calls for it. 

[ResolvingProjectAsPackageReference.zip](https://github.com/NuGet/Home/files/12059545/ResolvingProjectAsPackageReference.zip)